backend/src/main/modules/helper.py

```diff
@@ -0,0 +1,1010 @@
+import os
+import time
+from configparser import ConfigParser, ExtendedInterpolation
+import xlrd
+import uuid
+import csv
+from bson.objectid import ObjectId
+import json
+from datetime import datetime
+import requests
+from difflib import get_close_matches
+from requests import post, get, delete
+import sys
+import time
+import shutil
+from xlutils.copy import copy
+import shutil
+import re
+from xlrd import open_workbook
+from xlutils.copy import copy as xl_copy
+import logging.handlers
+import time
+from logging.handlers import TimedRotatingFileHandler
+import xlsxwriter
+import argparse
+import sys
+from os import path
+import openpyxl
+from openpyxl import Workbook, load_workbook
+from openpyxl.styles import Color, PatternFill, Font, Border
+from openpyxl.styles import colors
+from openpyxl.cell import Cell
+from common_config import *
+import threading
+
+millisecond = None
```

Comment:
millisecond is defined as a class variable also. 
I think this is a duplicate declaration. remove it from here and go with class assets. Test this after the change